rinja_derive/src/generator/expr.rs

```diff
@@ -676,6 +677,68 @@ impl<'a> Generator<'a, '_> {
         Ok(DisplayWrap::Unwrapped)
     }
 
+    fn _visit_value(
+        &mut self,
+        ctx: &Context<'_>,
+        buf: &mut Buffer,
+        args: &[WithSpan<'_, Expr<'a>>],
+        generics: &[WithSpan<'_, TyGenerics<'_>>],
+        node: Span<'_>,
+        kind: &str,
+    ) -> Result<DisplayWrap, CompileError> {
+        let [key] = args else {
+            return Err(ctx.generate_error(
+                format_args!("{kind} only takes one argument, found {}", args.len()),
+                node,
+            ));
+        };
+        let [gen] = generics else {
+            return Err(ctx.generate_error(
+                format_args!("{kind} expects one generic, found {}", generics.len()),
+                node,
+            ));
+        };
+        buf.write("rinja::helpers::get_value");
+        buf.write("::<");
+        self.visit_ty_generic(buf, gen);
+        buf.write('>');
+        buf.write("(&__rinja_values, &(");
+        self._visit_arg(ctx, buf, key)?;
+        buf.write("))");
+        Ok(DisplayWrap::Unwrapped)
+    }
+
+    fn _visit_ty(
+        &mut self,
+        ctx: &Context<'_>,
+        buf: &mut Buffer,
+        ty: &WithSpan<'_, Expr<'a>>,
+    ) -> Result<(), CompileError> {
+        match &**ty {
+            Expr::Var(ty) => {
+                buf.write(normalize_identifier(ty));
+            }
+            Expr::Path(ty) => {
+                self.visit_path(buf, ty);
+            }
+            Expr::Unary("&", rhs) => {
+                buf.write('&');
+                self._visit_ty(ctx, buf, rhs.as_ref())?;
+            }
+            Expr::BinOp(token @ ("<" | ">"), lhs, rhs) => {
+                self._visit_ty(ctx, buf, lhs.as_ref())?;
+                buf.write(token);
+                self._visit_ty(ctx, buf, rhs.as_ref())?;
+            }
+            _ => {
+                return Err(
+                    ctx.generate_error("expected a type name, not an expression", ty.span())
+                );
+            }
+        }
+        Ok(())
+    }
+
```

Comment:
```suggestion
```
The method is not used anymore.

Reply:
It's because it starts with an underscore. XD